### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by pdesai-crest**
Note:
In static failure,

min_number_log_statements - This is an expected failure, already discussed this with Matt
repo_name_has_expected_app_id and app_actions - ~~These failures will be passed after the app release MR will be merged~~ (It has been passed)

### CODE REVIEW

**Comment by mishalp-crest** [comment]
@AbrahamAjuelos, Can you please fix code-related linting and static tests failures like min_number_log_statements, product_name_on_files etc

**Comment by AbrahamAjuelos** [comment]
@mishalp-crest Can you provide some instructions on how to fix the following issues:

- min_number_log_statements: Not sure why it would complain when we have the relevant functions using save progress constantly.
- minimal data paths
- expected_app_id
- sequential_order
- verbose_periods

**Comment by mishalp-crest** [comment]
@AbrahamAjuelos 

For min_number_log_statements you can refer [this](https://github.com/splunk-soar-connectors/digitalshadows/pull/2/files#diff-bead98343fbbaf3c519e806bca093fd56ef379f6e24db4e472791d17e7dd0716R116)

I'll resolve other failures. 

Also kindly resolve linting failures too.

**Comment by AbrahamAjuelos** [comment]
@mishalp-crest 

Remaining tasks:

1. min_number_log_statements: the test will be updated from your side to pick subdirectory information. Noted as expected failure for now.

2. Remaining tests: minimal_data_paths, repo_name_has_expected_app_id and sequential_order. Can you help with these ones as we are unsure on what specifically needs to be actioned?

**Review by mmomin-crest — APPROVED** [review]

**Comment by mishalp-crest** [comment]
**Note:**
In static failure,
1. min_number_log_statements - This is an expected failure, already discussed this with Matt
2. repo_name_has_expected_app_id and app_actions - ~~These failures will be passed after the app release MR will be merged~~ (It has been passed)

File: darktrace.json [hunk 1]
```diff
@@ -0,0 +1,1193 @@
...
+            },
+            "read_only": true,
+            "render": {
+                "type": "table"
+            },
+            "type": "investigate",
+            "versions": "EQ(*)"
+        }
+    ],
+    "app_version": "1.0.2",
```
Comment (joshquerinphantom) [inline darktrace.json]: Would it be possible to re-order the .json file to have the app info at the top? I don't think it's strictly necessary, but would be nice. Specifically, move everything from line 1056 and below to the top. i.e.

- App info
- Asset config
- PIP wheels
- Action definitions, etc.

**Review by joshquerinphantom — CHANGES_REQUESTED** [review]

**Review by joshquerinphantom — APPROVED** [review]

**Pull request merged**